app/javascript/packs/MainApp/components/ProfileContent.jsx

```diff
@@ -27,12 +27,23 @@ class ProfileContent extends Component {
     onDismiss: PropTypes.func.isRequired
   }
 
-  renderNGBItem = ({ id, name, website }) => {
-    const showFlag = !(/(ireland|catalonia)/i.test(name))
+  getFlagForNGB = (name) => {
```

Comment:
Awesome function! Let's put this just above the class so it's not recalculated every time the class re-renders